.prow.yaml

```diff
@@ -355,6 +377,28 @@ postsubmits:
           requests:
             cpu: 2
             memory: 2Gi
+      - name: nephio-image-signer
```

Comment:
'- name:' field value has to be unique in whole config. Besides those jobs just generate images for test-infra use and we don't sign them, but I guess that's a good place to start.

Reply:
In all postsubmit jobs, `nephio-image-signer` is a separate container alongside `kaniko` within the same job definition. Since Prow jobs and k8s pods have similar specifications, and pods can contain multiple containers with unique names only within that specific pod, so why is the `- name:` field required to be unique across the entire configuration?